README.md

```diff
@@ -1,16 +1,52 @@
-![Cuprate](misc/logo/wordmark/CuprateWordmark.svg)
+<div align="center">
+	<img src="misc/logo/wordmark/CuprateWordmark.svg" width="50%"/>
 
+An alternative Monero node implementation.
 
-----
-
-> An up and coming Rust Monero node.
+_(work-in-progress)_
 
 [![Matrix](https://img.shields.io/badge/Matrix-Cuprate-white?logo=matrix&labelColor=grey&logoColor=white)](https://matrix.to/#/#cuprate:monero.social) [![CI](https://github.com/Cuprate/cuprate/actions/workflows/ci.yml/badge.svg)](https://github.com/Cuprate/cuprate/actions/workflows/ci.yml)
 
-Cuprate will be an alternative Monero node written from the ground up in Rust. It 
-will be able to independently validate Monero consensus rules providing a layer of 
-security and redundancy for the Monero network.
+</div>
+
+## Contents
+- [About](#about)
+- [Documentation](#documentation)
+- [Contributing](#contributing)
+- [Security](#security)
+- [License](#license)
+
+<!--
+TODO: add these sections someday.
+
+- [Status](#status) // when we're near v1.0.0
+- [Getting help](#getting-help) // issue tracker, user book, matrix channels, etc
+- [Build](#build)
+	- [Windows](#windows)
+	- [macOS](#macOS)
+	- [Linux](#Linux)
+
+-->
```

Comment:
This `README.md` is pretty sparse for now. We could add sections like `Goals` or `Status` but we'd have to update them frequently.

My vote is for just keeping it minimal until we have something working.

Reply:
I agree with keeping it minimal for now